packages/shared/src/store/assets/fungibleTokensSlice.ts

```diff
@@ -47,6 +47,10 @@ const fungibleTokensSlice = createSlice({
       .addCase(syncVerifiedFungibleTokens.pending, (state) => {
         state.loadingVerified = true
       })
+      .addCase(syncVerifiedFungibleTokens.rejected, (state) => {
+        state.loadingVerified = false
+        state.status = 'initialization-failed'
```

Comment:
The `initialization-failed` isn't used anywhere for now, right? I guess you added it so that we can show something in the UI at one point?

Reply:
https://github.com/alephium/alephium-frontend/blob/master/packages/shared/src/store/assets/assetsSelectors.ts#L28-L37

Before `initialization-failed` was introduced, the `status` was remaining `uninitialized`, which was causing the above selector to return `true`. By setting the `status` to `initialization-failed`, the selector returns now `false`.

Reply:
Ah!! Alright, makes sense.